lib/galaxy/authnz/custos_authnz.py

```diff
@@ -230,7 +230,7 @@ def _load_config_for_custos(self):
         # Set custos endpoints
         clientIdAndSec = self.config['client_id'] + ":" + self.config['client_secret']
         eps = requests.get(self.config['url'],
-                           headers={"Authorization": "Basic %s" % base64.b64encode(clientIdAndSec)},
+                           headers={"Authorization": "Basic %s" % base64.b64encode(util.smart_str(clientIdAndSec))},
```

Comment:
```suggestion
                           headers={"Authorization": "Basic %s" % util.unicodify(base64.b64encode(util.smart_str(clientIdAndSec)))},
```

Reply:
@nsoranzo In what situations is the unicodify necessary here?

Reply:
On Python3 always, otherwise this would result in:
`"Basic b'bWRma2Jua2RmYmRm'"` instead of `'Basic bWRma2Jua2RmYmRm'`

Reply:
^ yep, I saw it once I flipped python versions and tested.  Thanks!